config/routes.rb

```diff
@@ -1,13 +1,18 @@
 # frozen_string_literal: true
 Rails.application.routes.draw do
+  devise_for :users,
+             controllers: { omniauth_callbacks: 'users/omniauth_callbacks' },
+             skip: [:sessions]
+
   devise_scope :user do
-    get 'wp-admin', to: 'devise/sessions#new'
+    get 'wp-admin', to: 'devise/sessions#new', as: :new_user_session
```

Comment:
We were planning to change default logging URL for admins/moderators. It is present in already merged PR.